app/assets/javascripts/modules/media_viewer.js

```diff
@@ -98,42 +157,23 @@
           var checkWindow = setInterval(function() {
             if (!_timedOut(start, 30000) &&
               (!windowReference || !windowReference.closed)) return;
-            clearInterval(checkWindow);
-            mediaObject
-              .parents(sliderObjectSelector)
-              .find('[data-auth-link], ' + restrictedOverlaySelector + ', ' + restrictedMessageSelector)
-              .hide();
-            // As best I can tell, .touch() does nothing (and does not exist in jQuery),
-            // but without calling some function after .load() the media fails to load.
-            mediaObject[0].load().touch();
+            authCheckForMediaObject(mediaObject, function(_, data) {
+              if(data.status === 'success') {
+                clearInterval(checkWindow);
+                mediaObject
+                  .parents(sliderObjectSelector)
+                  .find('[data-auth-link], ' + restrictedOverlaySelector + ', ' + restrictedMessageSelector)
+                  .hide();
```

Comment:
sorry for the JS/JQuery naivete, but is what's being found and hidden here in this callback different than the elements that are being found and hidden in lines 128-134?  or is it happening at a different point in the process (e.g. before vs after login)?  couldn't tell whether this was duplicating behavior or if i was just misunderstanding things.